rosdep/python.yaml

```diff
@@ -2811,6 +2811,39 @@ python-tk:
     wily_python3: [python3-tk]
     xenial: [python-tk]
     xenial_python3: [python3-tk]
+python-torch-cuda75-pip:
+  debian:
+    pip:
+      packages: ['http://download.pytorch.org/whl/cu75/torch-0.1.11.post4-cp27-none-linux_x86_64.whl']
```

Comment:
> If you ask apt to install a package it will install the latest version available or upgrade that package if it's already installed.

Oh, I didn't know that. Thanks.